### PR TITLE
Spacy matcher into lambda

### PR DESCRIPTION
**Pull request by ANA-POTJE**

### CODE REVIEW

**Review by jsjohnstone — DISMISSED** [review]

**Comment by ANA-POTJE** [comment]
1) Lambda_function.py, including 'topics' in the JSON object (from spacyMatcher.py output data):
{
  "url": ...
  "results": ...
  "article": {
    "header":...
    "topics":[
      {"type":"PERSON", "topic":"Boris Johnson"},
      {"type":"GPE","topic":"United Kingdom"}
    ]
  }
}

2) Included in build_script.sh: => python -m spacy download en_core_web_sm
3) Updated spacyMatcher.py with "XRAY" calls for debug and included TAGS description.

**Review by jsjohnstone — APPROVED** [review]

File: src/spacyMatcher.py [hunk 1]
```diff
@@ -8,20 +8,33 @@
...
 ###########################################################################################################
 
-def spacyMatcher(url, tag): 
-    import getText as getTxt
-    #import spacy
-    import en_core_web_sm
+from aws_xray_sdk.core import xray_recorder
+
+@xray_recorder.capture('spacyMatcher')
+
```
Comment (jsjohnstone) [inline src/spacyMatcher.py:30]: Please remove this space

**Review by jsjohnstone — CHANGES_REQUESTED** [review]

File: src/spacyMatcher.py (hunk 1 above)
Comment (ANA-POTJE) [reply src/spacyMatcher.py]: Done ... Please check the code now ...

**Review by jsjohnstone — APPROVED** [review]

**Comment by ANA-POTJE** [comment]
Changes in spacyMatcher - it can now process all tags at once!
Please check the code ...

**Review by jsjohnstone — APPROVED** [review]

**Pull request merged**